tsconfig.base.json

```diff
@@ -39,7 +39,7 @@
     // "noUnusedParameters": true,            /* Report errors on unused parameters. */
     "noImplicitReturns": true,                /* Report error when not all code paths in function return a value. */
     "noFallthroughCasesInSwitch": true,       /* Report errors for fallthrough cases in switch statement. */
-
+    "skipLibCheck": true,
```

Comment:
hmmm... do you think this is safe? From what i read, it's good to not skip it until parts of your project stops compiling then you wack just that part out with this 🤔

Reply:
Updated to remove this